.eslintrc.js

```diff
@@ -1,24 +1,24 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint/eslint-plugin'],
-  extends: [
-    'plugin:@typescript-eslint/recommended',
-    'plugin:prettier/recommended',
-  ],
-  root: true,
-  env: {
-    node: true,
-    jest: true,
-  },
-  ignorePatterns: ['.eslintrc.js'],
-  rules: {
-    '@typescript-eslint/interface-name-prefix': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/explicit-module-boundary-types': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-  },
+  // parser: '@typescript-eslint/parser',
+  // parserOptions: {
+  //   project: 'tsconfig.json',
+  //   sourceType: 'module',
+  // },
+  // plugins: ['@typescript-eslint/eslint-plugin'],
+  // extends: [
+  //   'plugin:@typescript-eslint/recommended',
+  //   'plugin:prettier/recommended',
+  // ],
+  // root: true,
+  // env: {
+  //   node: true,
+  //   jest: true,
+  // },
+  // ignorePatterns: ['.eslintrc.js'],
+  // rules: {
+  //   '@typescript-eslint/interface-name-prefix': 'off',
+  //   '@typescript-eslint/explicit-function-return-type': 'off',
+  //   '@typescript-eslint/explicit-module-boundary-types': 'off',
+  //   '@typescript-eslint/no-explicit-any': 'off',
+  // },
```

Comment:
If this lint file is not required, could you please delete this file?

Reply:
> If this lint file is not required, could you please delete this file?

yeah again. i usually start uninstalling and deleting eslint since TS does all the work. I dont know why ppl use it in TS projects generally.